perf/run-benchmarks

```diff
@@ -453,18 +579,21 @@ def run_benchmarks(
     corpuses = SMALL_CORPUSES + MEDIUM_CORPUSES
     if dummy:
         corpuses = DUMMY_CORPUSES
-    if internal:
+    elif internal:
         corpuses = INTERNAL_CORPUSES
-    if gitlab:
+    elif gitlab:
         corpuses = GITLAB_CORPUSES
         variants = GITLAB_VARIANTS
-    if small_only:
+    elif small_only:
         corpuses = SMALL_CORPUSES
-    if all:
+    elif all:
         corpuses = SMALL_CORPUSES + MEDIUM_CORPUSES + LARGE_CORPUSES
+    elif config_file:
```

Comment:
should this be `elif not config_file is None`?

Reply:
On this line (https://github.com/returntocorp/semgrep/pull/3171/files#diff-903f08c7f7a15ccdd118e5e7e62172cac2dd2e20ce2b1d8bbd31bf75f6808994R763), the `config_file` parameter is either set to a `Path` or is `None`. Python evaluates `None` as `False`, so if `config_file` is `None` this will be skipped.